app/packs/src/apps/mydb/collections/CollectionSubtree.js

```diff
@@ -205,48 +162,51 @@ export default class CollectionSubtree extends React.Component {
     CollectionActions.updateCollectrionTree(newIds)
   }
 
-  synchronizedIcon() {
-    let sharedUsers = this.state.root.sync_collections_users
-    return (
-      sharedUsers && sharedUsers.length > 0
-        ? <OverlayTrigger placement="bottom" overlay={UserInfos({ users: sharedUsers })}>
-          <i className="fa fa-share-alt" style={{ float: "right" }}></i>
-        </OverlayTrigger>
-        : null
-    )
-  }
-
-
   render() {
-    const { fakeRoot } = this.props;
-    const { label, root } = this.state;
-    let { visible } = this.state;
+    const { label, root, visible, selected } = this.state;
+    const sharedUsers = root.sync_collections_users;
+    const children = this.children();
+
+    const showGatePushButton = root && root.is_locked && label === 'chemotion-repository.net';
 
-    let style;
-    if (!visible) {
-      style = {
-        display: 'none',
-        marginBottom: 0
-      };
-    }
-    const gated = root && root.is_locked && label == 'chemotion-repository.net' ?
-      <GatePushBtn collection_id={root.id} /> : null;
     return (
       <div className="tree-view" key={root.id}>
-        {this.takeOwnershipButton()}
-
-        <div id={`tree-id-${root.label}`} className={"title " + this.selectedCssClass()}
-          onClick={this.handleClick}>
-          {this.expandButton()}
-          {this.synchronizedIcon()}
-          {gated}
-          {label}
+        <div
+          id={`tree-id-${root.label}`}
+          className={`title ${selected ? 'selected' : ''} d-flex align-items-baseline`}
+          onClick={this.handleClick}
+        >
+          {showGatePushButton && (<GatePushBtn collection_id={root.id} />)}
+          <span className="me-auto">{label}</span>
+          {this.canTakeOwnership() && (
+            <i
+              className="fa fa-exchange"
+              onClick={e => this.handleTakeOwnership(e)}
```

Comment:
handleTakeOwnership takes no parameters, so you could just use
```javascript
onClick={this.handleTakeOwnership}
```

Reply:
onClick={this.handleTakeOwnership} is better